samples/streamlit-in-snowflake/sis-deploy/README.md

```diff
@@ -1,10 +1,15 @@
-# Dependencies
+# Test and Deploy apps with the Snowcli
 
-pip install snowflake-cli-labs==1.2.1
+Often when developing SiS apps, you might want to graduate from the Snowsight UI and begin to develop locally with your preferred development environment (like VSCode), or collaborate with your team on GitHub or another git based environment. This repo provides an example of how to:
+- Set up your Streamlit and Snowflake credentials
+- Install the right dependencies
+- Test and deploy your SiS apps from the command line
+
+If you have issues with this example, feel free to open an issue, talk to your SE, or email me at tyler.richards@snowflake.com. You can find full documentation for the snowcli [here](https://github.com/Snowflake-Labs/snowcli) and specific documentation for the snowcli and SiS [here](https://docs.snowflake.com/LIMITEDACCESS/snowcli/streamlit-apps/overview).
```

Comment:
Are we supposed to send out LIMITEDACCESS links?

Reply:
yeah it should be fine